GPU/Common/SplineCommon.cpp

```diff
@@ -223,6 +224,13 @@ static void spline_knot(int n, int type, float *knot) {
 	}
 }
 
+bool CanUseHardwareTessellation(GEPatchPrimType prim) {
+	if (g_Config.bHardwareTessellation && !g_Config.bSoftwareRendering) {
+		return CanUseHardwareTransform(PatchPrimToPrim(prim));
+	}
```

Comment:
We can write more simply, but it's just a matter of taste.
```
bool CanUseHardwareTessellation(GEPatchPrimType prim) {
	return g_Config.bHardwareTessellation && !g_Config.bSoftwareRendering && CanUseHardwareTransform(PatchPrimToPrim(prim));
}
```

Reply:
I didn't want the line to be overly long.

-[Unknown]